main.py

```diff
@@ -809,7 +813,7 @@ def create_years_list():
         try:
             year_col = db.session.query(distinct(func.extract("year", Marker.created)))
             break
-        except OperationalError:
+        except:
```

Comment:
I remember it was just throwing an `OperationalError` sometimes due to a locked DB, which is why I wrote this loop to retry the query until success. I don't think it was an SQLAlchemy exception.